docs/propose_changelog.py

```diff
@@ -13,7 +13,7 @@
 
 # config
 #          see: https://github.com/settings/tokens
-g = Github("add token with public repo read rights here")
+g = Github("49265fe2d70063570d92fbabc0c966837d0062bc")
```

Comment:
uhoh... this please:
- revert this change
- quickly remove this token in your settings, this is a security leak with access to your account:
https://github.com/settings/tokens

Reply:
thx deleted the token. I will do an update for this tool to read the token from a file in home.

Reply:
update: token deleted